package/kernel/linux/modules/hwmon.mk

```diff
@@ -77,6 +77,22 @@ endef
 $(eval $(call KernelPackage,hwmon-adt7475))
 
 
+define KernelPackage/hwmon-dme1737
+  TITLE:=SMSC DME1737 and compatible monitoring support
+  KCONFIG:=CONFIG_SENSORS_DME1737
+  FILES:= \
+	$(LINUX_DIR)/drivers/hwmon/dme1737.ko
+  AUTOLOAD:=$(call AutoProbe,dme1737)
+  $(call AddDepends/hwmon,+kmod-i2c-core +kmod-i2c-piix4)
```

Comment:
+kmod-i2c-piix4

Reply:
@hauke had the false impression that is needed,  verified it and removed